src/elm/Page/Transfer.elm

```diff
@@ -0,0 +1,414 @@
+                            [ div [ class "border border-solid border-green border-t-0 border-r-3 border-b-3 border-l-0 inline-block p-1 -rotate-45 -ml-12 mt-5" ]
+                                []
+                            ]
+                        ]
+    in
+    div [ class "-ml-16 flex flex-row mt-5" ]
+        [ head
+        , div [ class "px-4 py-2 m-2" ]
+            [ div [ class "-ml-10 border border-solid rounded border-green bg-white" ]
+                [ div [ class "ml-1" ]
+                    [ p [ class "pt-1 text-caption font-hairline text-gray-900 pl-1" ]
+                        [ text <|
+                            case state of
+                                Received ->
+                                    String.toUpper (t "transfer_result.received")
+
+                                Transferred ->
+                                    String.toUpper (t "transfer_result.transferred")
+
+                                NotInvolved ->
+                                    String.toUpper (t "transfer_result.transferred")
+                        ]
+                    , div [ class "flex flex-row" ]
+                        [ p [ class "pl-1 pr-5 font-medium text-green" ]
+                            [ text <|
+                                String.fromFloat transfer.value
+                            ]
+                        , span [ class "ml-2 text-caption text-green mt-1 mb-1 font-thin pr-3 pl-3" ]
+                            [ text <| Eos.symbolToString transfer.symbol ]
+                        ]
+                    ]
+                ]
+            ]
+        , tail
+        ]
+
+
+viewCommunity : LoggedIn.Model -> Transfer -> Html Msg
+viewCommunity { shared } transfer =
+    let
+        t str =
+            I18Next.t shared.translations str
+                |> String.toUpper
+
+        date =
+            Just transfer.blockTime
+                |> Utils.posixDateTime
+                |> Strftime.format "%d %b %Y" Time.utc
+    in
+    div [ class "flex mb-4 bg-white" ]
+        [ div [ class "w-full h-50 mt-20 mb-10" ]
+            [ viewRest (t "transfer_result.community") <| Eos.symbolToString transfer.symbol
+            , viewRest (t "transfer_result.date") date
+            , viewRest (t "transfer_result.message") <| Maybe.withDefault "" transfer.memo
+            ]
+        ]
+
+
+viewRest : String -> String -> Html Msg
+viewRest title content =
+    div [ class "mt-5 ml-16" ]
+        [ h5 [ class "leading-tight text-xs mb-1 text-reward-green" ]
+            [ text title ]
+        , p [ class "text-lg font-sans not-italic" ]
+            [ text content ]
+        ]
+
+
+
+-- UPDATE
+
+
+type Msg
+    = CompletedTransferLoad (Result (Graphql.Http.Error (Maybe Transfer)) (Maybe Transfer))
+
+
+type alias UpdateResult =
+    UR.UpdateResult Model Msg (External Msg)
+
+
+findState : Maybe Transfer -> LoggedIn.Model -> State
+findState maybeTransfer { accountName } =
```

Comment:
Elegant solution to this! I like it a lot!

Reply:
The pancake solution